src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.cs

```diff
@@ -517,13 +517,14 @@ private HttpClient CreateHttpClient()
                 {
                     httpClientHandler.Proxy = _httpConnectionOptions.Proxy;
                 }
-                if (_httpConnectionOptions.Cookies != null)
+
+                // Only access HttpClientHandler.ClientCertificates and HttpClientHandler.CookieContainer
+                // if the user has configured client certs
+                // Some skews of Mono do not support client certs or cookies and will throw NotImplementedException
```

Comment:
```suggestion
                // Some variants of Mono do not support client certs or cookies and will throw NotImplementedException
```

a) it's "SKUs" not "skews" but..
b) "SKUs" is a wonky internal term, might as well just say variants :)